src/shared/utils/isDateISOString.ts

```diff
@@ -0,0 +1,2 @@
+export const isDateISOString = (dateString: string) =>
```

Comment:
It's used in the `validations` file in this PR to generate zod rules for both a date and a range of dates.
I think it would be a nice addition to the boilerplate to have a standard on date format and ISO seems like a good choice.